packages/core/src/lib/endpoint-version.ts

```diff
@@ -0,0 +1,32 @@
+import { isNode } from '@lit-protocol/misc';
+import { LitEndpoint } from '@lit-protocol/types';
+
+/**
+ * Compose the Lit URL
```

Comment:
can we add a bit more context in this comment on our versioning schema in our routing? Meaning how we are adding `vx` where `x` is the version number.